src/pages/ManageTransactions/index.js

```diff
@@ -12,12 +18,20 @@ const ManageTransactions = () => {
         filteredTransactions={filteredTransactions}
         setFilteredTransactions={setFilteredTransactions}
       />
-      <StyledManageTransactions>
-        <TransactionList
-          text="No transactions with the selected parameters"
-          transactions={filteredTransactions}
-        />
-      </StyledManageTransactions>
+      <StyledStatisticsWrapper>
+        <StyledGraphWrapper>
+          {filteredTransactions.length
```

Comment:
this conditional rendering should surround the very start of rendering anything related to the graph (<StyledStatisticsWrapper>)

also you can use && operator there.. instead of ? : null